hooks/tests/run_all_hook_tests.sh

```
@@ -0,0 +1,77 @@
+TOTAL_FAILED=0
+FAILED_TESTS=()
+
+echo -e "${BLUE}╔════════════════════════════════════════════╗${NC}"
+echo -e "${BLUE}║     Claude Code Hooks Test Suite          ║${NC}"
+echo -e "${BLUE}╚════════════════════════════════════════════╝${NC}"
+echo ""
+
+# Function to run individual test files
+run_test_file() {
+    local test_file="$1"
+    local test_name=$(basename "$test_file" .sh)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)